src/EnergyPlus/DXCoils.cc

```diff
@@ -15223,7 +15223,7 @@ void CalcTwoSpeedDXCoilStandardRating(EnergyPlusData &state, int const DXCoilNum
                 Real64 TotCapTempModFac = CurveManager::CurveValue(state, coil.CCapFTemp(1), wbRated, par3);
                 Real64 HighSpeedNetCoolingCap = coil.RatedTotCap(1) * TotCapTempModFac * TotCapFlowModFac - FanHeatCorrection;
 
-                TotCapFlowModFac = CurveManager::CurveValue(state, coil.CCapFFlow(1), AirMassFlowRatio);
+                // TotCapFlowModFac = CurveManager::CurveValue(state, coil.CCapFFlow(1), AirMassFlowRatio);
                 TotCapTempModFac = CurveManager::CurveValue(state, coil.CCapFTemp2, wbRated, par3);
```

Comment:
This is another place where the line of code is redundant. This function call is not needed, as it was already evaluated four lines above (at Line 15222), and the value has not been changed since.

Reply:
@jcyuan2020 Do you understand why a local scope `TotCapFlowModFac` is defined in this block instead of using the function-level reference from line 14951? I'm surprised the compiler doesn't complain about a redefinition.

In general, I don't understand most of those references here in this function since the values all appear to be used locally for the standard rating calculation.

Also, if I'm following correctly, this fix only impacts the standard rating results. That should be made clear in the top comments of the PR.

Reply:
> In general, I don't understand most of those references here in this function since the values all appear to be used locally for the standard rating calculation.

Ignore that question. I see there's a call to `CalcMultiSpeedDXCoil` which may need these referenced values.

Reply:
@mjwitte For the scope question: I think it is valid to have the same name for the two variables of different scopes (although might not be a good practice in general). The more "local" one will take precedence in this case---the "local" scope will end at Line 15244 and before that, the more "local" will used by the compiler. So theoretically there should be no ambiguity to the compiler here.

So here if this should changed to the functional really depends on the author's original intent---it seems reasonable if the author's original intent is to keep this part local according to what happened before Line 15244. It is validly used locally and then destroyed automatically at the end of Line 15244. The wider-scoped functional one will not be affected.

For clarity purpose, the "local" ones could be renamed. But even without renaming the code and logic are still valid, though.